src/Data/Machine/Mealy.hs

```diff
@@ -205,6 +209,10 @@ instance Closed Mealy where
 instance Semigroup b => Semigroup (Mealy a b) where
   f <> g = Mealy $ \x -> runMealy f x <> runMealy g x
 
+#if MIN_VERSION_base(4,11,0)
 instance Monoid b => Monoid (Mealy a b) where
+#else
```

Comment:
The `else` branch looks strictly worse than what we had before. This is an unforced breaking change. Let's have two compete instances instead, which will give each exactly what it needs and wants.